src/main/java/internet/shop/controllers/drivers/GetMyCurrentCarsController.java

```diff
@@ -22,7 +22,7 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp)
         List<Car> carList = carService.getAllByDriver((Long) req
                 .getSession().getAttribute(DRIVER_ID));
         req.setAttribute("cars", carList);
-        req.getRequestDispatcher("/WEB-INF/views/drivers/cars_by_driver_id.jsp")
+        req.getRequestDispatcher("/WEB-INF/views/drivers/get.jsp")
```

Comment:
тут точно все працює? можливо так:
```suggestion
        req.getRequestDispatcher("/WEB-INF/views/cars/get.jsp")
```

Reply:
done